README.md

```diff
@@ -2,40 +2,55 @@
+
 
 ## Baseline
-- Create a `Planet` class with a name attribute.
-- You should be able to instantiate a new `Planet` object with an associated name.
+- Create a `SolarSystem` class with an `@planets` instance variable.
+- The initialize method should take a collection of planet names and store them in `@planets`
+- Create a method to add a planet to the list.
+- Create a method which will return a list of the planets in this style:
+
+```bash
+1.  Mercury
+2.  Venus
+3.  Earth
+4.  Mars
+5.  Jupiter
+6.  Hoth
+```
+
+
 
 # Wave 1
 ## Primary Requirements
-- Get creative! Give each instance of `Planet` at least five attributes. These could be diameters, mass, moons... whatever!
-- Allow these attributes to be set using a hash in `initialize`.
-- You should be able to create many different planets with different properties, like Mercury, Venus, Earth, Mars, Jupiter, etc.
+- Get creative! Instead of Strings for planets, give SolarSystem a list of hashes where each planet is sent as a hash with at least 5 attributes.  
 
 ## Optional Enhancements
 - Give each planet a rate of solar rotation
-- Give each planet a `@distance_from_the_sun` attribute
+- Give each planet a `distance_from_the_sun` attribute
 - Write a program that asks for user input to query the planets:
   - First, ask the user to select a planet they'd like to learn about.
-  - Present the user with a list of planets from which they can choose. Something like:
+  - Use your method to present the user with a list of planets from which they can choose. Something like:
     - `1. Mercury, 2. Venus, 3. Earth, 4. Secret Earth, 5. Mars, 6. Jupiter, ... 13. Exit`
   - Provide the user with well formatted information about the planet (diameter, mass, number of moons, primary export, etc.)
   - Then ask the user for another planet.
 
 # Wave 2
 ## Primary Requirements
-- Create a `SolarSystem` class that has an attribute `planets` that has zero to many `Planet` instances. There are a few different options for how to associate the planets with your solar system:
-    - Create a method that adds a single planet to a solar system
-    - Create a method that adds a list of planets to an existing list of planets
+- Create a `Planet` class which will represent a planet.
+    - Add an `initialize` method which takes a hash and uses the hash to set the class' instance variables.  
+    - Create a method that **returns** the Planet's attributes in an easy to read fashion.
+    - Create accessor methods to the instance variables.
+
+# Wave 3
+- Modify your `SolarSystem` class to use the Planet class instead of a hash.
+- Create an interface where the user can select a planet and be presented information about the planet.  
+- Allow your user to add their own planet.  
 
 ## Optional Enhancements
 - Ensure that the each planet has a `@distance_from_the_sun` attribute. Using this data, add a method to determine the distance from any other planet (assuming planets are in a straight line from the sun)
```

Comment:
Jamie and I got a lot of questions about what was meant by "formation year" - maybe changing this to "age" would be easier to understand.